common/flags/Flags.java

```diff
@@ -110,21 +110,17 @@ static FlagData getFlag(String name) {
 
   @VisibleForTesting
   static void setFlagValue(String name, String value) {
-    instance().flags.put(name, getFlag(name).toBuilder().setValue(value).setHasValue(true).build());
+    instance().flags.put(name, getFlag(name).toBuilder()
+        .setValue(value
+            .replace("[", "")
```

Comment:
Looks like we're removing `[` and `]` and spaces before commas from all flags. What if I want to pass a string flag that has `[`?